lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -815,7 +816,33 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :public => true,
           :type => Boolean,
           :allowed_from_server => false,
-          :description => 'If `true`, disables agent middleware for Sinatra. This middleware is responsible for advanced feature support such as [cross application tracing](/docs/apm/transactions/cross-application-traces/cross-application-tracing), [page load timing](/docs/browser/new-relic-browser/getting-started/new-relic-browser), and [error collection](/docs/apm/applications-menu/events/view-apm-error-analytics).'
+          :description => 'If `true`, disables agent middleware for Sinatra. This middleware is responsible for advanced feature support such as [cross application tracing](/docs/apm/transactions/cross-application-traces/cross-application-tracing), [page load timing](/docs/browser/new-relic-browser/getting-started/new-relic-browser), and [error collection](/docs/apm/applications-menu/events/view-apm-error-analytics).'\
+          "\n" \
+          "\n" \
+          '    <Callout variant="important">' \
+          "\n" \
+          '      Cross application tracing is deprecated in favor of [distributed tracing](https://docs.newrelic.com/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing). Distributed tracing is on by default for Ruby agent versions 8.0.0 and above. Middlewares are not required to support distributed tracing.' \
+          "\n" \
+          "\n" \
+          '      To continue using cross application tracing, update the following options in your `newrelic.yml` configuration file:'\
+          "\n" \
+          "\n" \
+          '      ```'\
+          "\n" \
+          '      # newrelic.yml'\
+          "\n" \
+          "\n" \
+          '        cross_application_tracer:'\
+          "\n" \
+          '          enabled: true'\
+          "\n" \
+          '        distributed_tracing:'\
+          "\n" \
+          '          enabled: false'\
+          "\n" \
+          '      ```'\
+          "\n" \
+          '    </Callout>'
```

Comment:
This multiline text might be a good candidate for a Heredoc, so that you can skip all of the `\n`, `'`, and `\` characters.

```ruby
          description => <<~DESCRIPTION
          If `true`, disables agent middleware for Sinatra. This middleware is responsible for advanced feature support such as [cross application tracing](/docs/apm/transactions/cross-application-traces/cross-application-tracing), [page load timing](/docs/browser/new-relic-browser/getting-started/new-relic-browser), and [error collection](/docs/apm/applications-menu/events/view-apm-error-analytics).


              <Callout variant="important">

                Cross application tracing is deprecated in favor of [distributed tracing](https://docs.newrelic.com/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing). Distributed tracing is on by default for Ruby agent versions 8.0.0 and above. Middlewares are not required to support distributed tracing.


                To continue using cross application tracing, update the following options in your `newrelic.yml` configuration file:


                ```

                # newrelic.yml


                  cross_application_tracer:

                    enabled: true

                  distributed_tracing:

                    enabled: false

                ```

              </Callout>
          DESCRIPTION
```

Reply:
Huh, so I updated all of the longer descriptions to use heredoc like that, but it appears to not be supported by ruby 2.2. I remember you mentioning that there were a couple different versions of heredoc that came out at different times, so I'll look and see what works on 2.2